src/textual/widgets/_list_view.py

```diff
@@ -262,7 +262,7 @@ def remove_items(self, indices: Iterable[int]) -> AwaitRemove:
         for index in indices:
             items_to_remove.append(items[index])
 
-        await_remove = items.remove()
+        await_remove = self.app._prune(*items_to_remove, parent=self)
```

Comment:
I wanted to avoid this, because widgets should be able to be written with the public API only.

Ideally we can do this through a DOMQuery, although there isn't an elegant way of excluding widgets (just selectors).

One possibility that comes to mind is that DOMQuery.remove grows an `exclude_widgets` parameter, but I am open to suggestions.

Reply:
Tom, hope you don't mind, I'm going to make that change to this PR